docs: added a info panel reg. Enable monitoring on thin pool (#457)
openebs/website: docs/main/user-guides/local-storage-user-guide/local-pv-lvm/advanced-operations/lvm-thin-provisioning.md

## Configure Auto Extending of the Thin Pool (Configure Over-Provisioning protection)

:::important

> The provisioner does not automatically enable monitoring (auto-extend) on the thin pool, as not all use cases necessitate auto-extension. In certain configurations, a thin pool may be necessary to extend, but it must not exceed a certain limit. However, the lvm configuration does not offer a method to restrict the extension limit.

Reviewer: ```suggestion
The provisioner does not automatically enable monitoring (auto-extend) on the thin pool, as not all use cases necessitate auto-extension. In certain configurations, a thin pool may be required to extend, but not exceeding a certain limit. However, the lvm configuration does not offer a method to restrict the extension limit.
```
